Lib/test/test_configparser.py

```diff
@@ -739,6 +740,18 @@ def test_read_returns_file_list(self):
         cf = self.newconfig()
         parsed_files = cf.read([])
         self.assertEqual(parsed_files, [])
+        # check when passing an existing bytestring path
```

Comment:
`test_read_returns_file_list` is already too long, could you move it a separate test case?